tests/validation/tests/v3_api/scripts/pipeline/Jenkinsfile_custom_cluster_matrix

```diff
@@ -111,19 +97,22 @@ node {
               // get the list of versions
               def versions = RANCHER_CLUSTER_OS_DOCKER_VERSION.split(',')
               for(int i = 0; i < versions.size(); i++) {
-                def version = versions[i].trim()
+                def items = versions[i].split(':')
+                def name = items[0].trim()
+                def ami  = items[1].trim()
+                def user = items[2].trim()
                 def params = [
                       string(name: 'CATTLE_TEST_URL', value: "${CATTLE_TEST_URL}"),
                       string(name: 'ADMIN_TOKEN', value: "${ADMIN_TOKEN}"),
                       string(name: 'USER_TOKEN', value: "${USER_TOKEN}"),
-                      string(name: 'RANCHER_SERVER_VERSION', value: "${rancher_version}"), 
                       string(name: 'BRANCH', value: branch),
-                      string(name: 'RANCHER_OS_DOCKER_VERSION', value: "${version}"),
+                      string(name: 'AWS_AMI', value: "${ami}"),
+                      string(name: 'AWS_USER', value: "${user}"),
                       string(name: 'DOCKER_INSTALLED', value: "${DOCKER_INSTALLED}"),
                       string(name: 'RANCHER_K8S_VERSION', value: "${env.RANCHER_K8S_VERSION}")
                       ]
                 println "parameters: ${params}"
-                jobs["${version}"] = { build job: 'rancher-v3_ontag_custom_certification', parameters: params }
+                jobs["${name}"] = { build job: 'jiaqi_rancher-v3_ontag_custom_certification', parameters: params }
```

Comment:
The job has the name jiaqi_rancher. It should be renamed to rancher-v3_ontag

Reply:
solved